app/views/user_mailer/idv_please_call.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= image_tag(
-      asset_url('email/phone_icon.png'),
+      attachments['phone_icon.png'].url,
```

Comment:
Question for my own education: what's the benefit of using this approach instead of the asset url?

Reply:
Many mail clients will not display images linked by URL by default (since they can be used to track folks opening emails).

Reply:
Ahhh, got it, thanks. 🙏🏻